models/config.py

```diff
@@ -187,4 +188,5 @@ def asSteamGame(self, steamID):
             return
         fn(self)
         self.vLaunch = f"steam://run/{steamID}"
+        self.vIcon = f"https://cdn.cloudflare.steamstatic.com/steam/apps/{steamID}/capsule_184x69.jpg"
```

Comment:
I like this. Adds a nice touch

Reply:
I cannot get the square icon URLs from steam programmatically. I would love some help. Until then we're stuck with these 'capsules'.